public/docs/ts/latest/guide/architecture.jade

```diff
@@ -392,151 +438,160 @@ figure
   Angular uses dependency injection to provide new components with the services they need.
 <br clear="all">
 :marked
-  In TypeScript, Angular can tell which services a component needs by looking at the types of its constructor parameters.
-  For example, the constructor of our `HeroListComponent` needs the `HeroService`:
-+makeExample('architecture/ts/app/hero-list.component.ts', 'ctor', 'app/hero-list.component (constructor)')(format=".")
+  Angular can tell which services a component needs by looking at the types of its constructor parameters.
+  For example, the constructor of our `HeroListComponent` needs a `HeroService`:
+
++makeExcerpt('app/hero-list.component.ts (constructor)', 'ctor')
+
 :marked
-  When Angular creates a component, it first asks an **Injector** for
-  the services that the component requires. 
-  
-  An `Injector` maintains a container of service instances that it has previously created.
+  When Angular creates a component, it first asks an **injector** for
+  the services that the component requires.
+
+  An injector maintains a container of service instances that it has previously created.
   If a requested service instance is not in the container, the injector makes one and adds it to the container
-  before returning the service to Angular. 
-  When all requested services have been resolved and returned, 
+  before returning the service to Angular.
+  When all requested services have been resolved and returned,
   Angular can call the component's constructor with those services as arguments.
   This is what we mean by *dependency injection*.
-  
+
   The process of `HeroService` injection looks a bit like this:
 figure
   img(src="/resources/images/devguide/architecture/injector-injects.png" alt="Service" )
 :marked
-  If the `Injector` doesn't have a `HeroService`, how does it know how to make one?
-  
-  In brief, we must have previously registered a **provider** of the `HeroService` with the `Injector`.
+  If the injector doesn't have a `HeroService`, how does it know how to make one?
+
+  In brief, we must have previously registered a **provider** of the `HeroService` with the injector.
   A provider is something that can create or return a service, typically  the service class itself.
-  
+
   We can register providers at any level of the application component tree.
   We often do so at the root when we bootstrap the application so that
   the same instance of a service is available everywhere.
-+makeExample('architecture/ts/app/main.ts', 'bootstrap','app/main.ts (excerpt)')(format=".")
+  
++makeExcerpt('app/main.ts', 'bootstrap')
+
 :marked
-  Alternatively, we might register at a component level ...
-+makeExample('architecture/ts/app/hero-list.component.ts', 'providers','app/hero-list.component.ts (excerpt)')(format=".")
+  Alternatively, we might register at a component level:
+
++makeExcerpt('app/hero-list.component.ts', 'providers')
```

Comment:
I think we're losing the point by including to much code.

Why not say "Alternatively, we might register at a component level in the `providers` property of the component metadata:"

Then the only thing in the "providers region" of the sample is:
```
  // #docregion providers
  providers:   [HeroService]
  // #enddocregion providers
```
```